### PR TITLE
♻️✨ FrequentMutations Relay containers

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

File: modules/node_modules/@ncigdc/containers/FrequentMutationsTable.js [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+  }
+  const frequentMutations = mapData(ssms.hits.edges.map(x => x.node), projectId);
+  return (
+    <div>
+      <Row style={{ backgroundColor: 'white', padding: '1rem', justifyContent: 'space-between' }}>
+        <Showing
+          docType="mutations"
+          prefix={'fmTable'}
+          params={relay.route.params}
+          total={ssms.hits.total}
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/containers/FrequentMutationsTable.js]: A little bit of weirdness with this 
image
I think it just needs to use `parseInt` for the size/offset inside Showing.

**Comment by Jephuff** [comment]
We should make sure these changes get made here too :) 
https://github.com/NCI-GDC/portal-ui/pull/1330
https://github.com/NCI-GDC/portal-ui/pull/1341

**Comment by cy** [comment]
if you merge this I can redo #1341

**Comment by alex-wilmer** [comment]
@cy it's okay I put it in the table for now. if we ever have the chart actually on its own we can deal with that then.

**Comment by cy** [comment]
oki, thought this looked pretty weird though
screen shot 2017-03-08 at 3 18 49 pm
so reusing #1341 to hide the chart

**Comment by alex-wilmer** [comment]
ooopps maybe just `if (!ssms.hits.length) return null`?

**Pull request merged**